great_expectations/datasource/fluent/pandas_datasource.py

```diff
@@ -80,7 +86,7 @@ def _get_reader_method(self) -> str:
         )
 
     def test_connection(self) -> None:
-        pass
+        ...
```

Comment:
@NathanFarmer Quick question -- what is the reason for the ellipsis?  Is there a `test_connection()` with a different signature that this method falls down to?  Thanks.

Reply:
The ellipsis can be thought of as the same as `pass` when it is used like this. I just like how it looks.